herbert_core/utilities/geometry/normal4D.m

```diff
@@ -0,0 +1,46 @@
+function orth = normal4D(three_vector)
+% function calculates the vector, orthogonal to 4-D plain defining by 3
+% input vectors.
+%
+% Input: 
+% three_vector -- 4x3 or 4x4 array defining three non-parallel vectors lying in
+%                 hyperplain in 4D. The x,y,z,t coordinates of each vector
+%                 are propagating along first dimension (column).
+%                 if there are 4 input vectors, they define the hyperplain,
```

Comment:
The function as defined operations on a 4x4 array -- it's not clear how the 'fake 4th dimension' is defined if the input is 4x3, or how that leads to a valid output.

Reply:
I have clarified it as I understand it